tidy3d/components/monitor.py

```diff
@@ -638,10 +645,23 @@ class ModeMonitor(AbstractModeMonitor):
         "and is hard-coded for other monitors depending on their specific function.",
     )
 
+    direction: Direction = pydantic.Field(
+        "+",
+        title="Propagation Direction",
+        description="Direction of propagation along the normal axis of the monitor used for the "
+        "computed mode field profiles.",
+    )
+
     def storage_size(self, num_cells: int, tmesh: int) -> int:
         """Size of monitor storage given the number of points after discretization."""
         # stores 3 complex numbers per frequency, per mode.
```

Comment:
comment is deprecated, or maybe add a similar comment about the fields too.

Reply:
Comments don't age well and can't be tested, so I removed it because I think the code is simple enough to document itself here.